docs/api/covidcast-signals/quidel.md

```diff
@@ -1,12 +1,141 @@
 ---
 title: Quidel
-parent: Inactive Signals
+parent: Data Sources and Signals
 grand_parent: COVIDcast API
 ---
 
 # Quidel
+{: .no_toc}
 
 * **Source name:** `quidel`
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+## COVID-19 Tests
+
+* **First issued:** 27 July 2020 
+* **Number of data revisions since 19 May 2020:** 0
+* **Date of last change:** Never
+* **Available for:** hrr, msa, state (see [geography coding docs](../covidcast_geography.md))
+
+Data source based on COVID-19 Antigen tests, provided to us by Quidel, Inc. When
+a patient (whether at a doctor’s office, clinic, or hospital) has COVID-like
+symptoms, doctors may order an antigen test. An antigen test can detect parts of
+the virus which are present during an active infection. This is in contrast with
```

Comment:
```suggestion
the virus that are present during an active infection. This is in contrast with
```
One of my thesis committee members corrected me on this repeatedly and now I can't not correct it